rdkit/Chem/Draw/MolDrawing.py

```diff
@@ -84,19 +74,15 @@ class DrawingOptions(object):
 
 
 class MolDrawing(object):
-  atomPs = None
-  canvas = None
-  canvasSize = None
```

Comment:
These are object properties and not class properties and therefore should be in __init__